src/pages/Home.jsx

```diff
@@ -1,5 +1,117 @@
+import React, { useState, useCallback } from 'react';
+import { Link } from 'react-router-dom';
+import Input from '../components/Form/Input';
+import { Button } from '@headlessui/react';
+import cover from '../assets/images/cover.png';
+import books from '../assets/images/books-home.png';
+import booksIcon from '../assets/images/icons/books.png';
+import handIcon from '../assets/images/icons/hand.png';
+import personIcon from '../assets/images/icons/person-reading.png';
+import arrowRight from '../assets/images/icons/arrow-right.svg';
+import BooksList from '../components/Books/BooksList';
+
 const Home = () => {
-  return <>Home page</>;
+  const [author, setAuthor] = useState('');
+  const [title, setTitle] = useState('');
+  const [isbn, setIsbn] = useState('');
+  const booksList = [
+    { image: cover, title: 'Karlsson on the Roof', author: 'Astrid Lindren' },
+    { image: cover, title: 'Book 2', author: 'Author 2' },
+    { image: cover, title: 'Book 3', author: 'Author 3' },
+  ];
+
+  const handleAuthorChange = useCallback((event) => {
+    setAuthor(event.target.value);
+  }, []);
+
+  const handleTitleChange = useCallback((event) => {
+    setTitle(event.target.value);
+  }, []);
+  const handleIsbnChange = useCallback((event) => {
+    setIsbn(event.target.value);
+  }, []);
+
+  return (
+    <>
+      <div className="flex size-full flex-col items-center justify-center gap-8 p-6">
+        <img className="h-56 w-64" src={books} alt="books" />
+        <form className="flex size-full flex-col bg-lightBlue px-8 py-12">
+          <h1 className="mb-6 font-headings text-2xl font-bold">
+            Search for books
+          </h1>
+
+          <Input
+            id="author"
+            name="author"
+            type="text"
+            value={author}
+            onChange={handleAuthorChange}
+          >
+            Author
+          </Input>
+          <Input
+            id="title"
+            name="title"
+            type="text"
+            value={title}
+            onChange={handleTitleChange}
+          >
+            Title
+          </Input>
+          <Input
+            id="isbn"
+            name="isbn"
+            type="text"
+            value={isbn}
+            onChange={handleIsbnChange}
+          >
+            ISBN
+          </Input>
+          <Button
+            as="button"
+            type="submit"
+            className="mt-7 w-full rounded-md bg-darkGreen p-2 font-semibold tracking-wide text-white"
```

Comment:
You can add it

`transition-transform duration-150 hover:bg-darkGreenHover active:scale-95`